Source/SessionDelegate.swift

```diff
@@ -299,7 +299,7 @@ extension SessionDelegate: URLSessionDownloadDelegate {
 
             request.didFinishDownloading(using: downloadTask, with: .success(destination))
         } catch {
-            request.didFinishDownloading(using: downloadTask, with: .failure(error))
+            request.didFinishDownloading(using: downloadTask, with: .failure(AFError.downloadedFileMoveFailed(error: error, destination: destination)))
```

Comment:
Looking at this usage site, I think we may want to capture the source `URL` as well as the destination, for better error descriptions:
```
AFError.downloadedFileMoveFailed(source: location, destination: destination, error: error)
```

Reply:
Done.